src/plugins/intel_gpu/src/kernel_selector/jitter.cpp

```diff
@@ -563,7 +563,7 @@ JitDefinitions DataTensorJitConstant::GetDefinitions() const {
             definitions.push_back({ safe_index_func_name, "0" });
             definitions.push_back({ index_func_name, "0" });
         }
-    } else if (_tensor.LogicalSize() == _tensor.Feature().v && !_tensor.is_dynamic()) {
+    } else if (_tensor.LogicalSize() == _tensor.Feature().v && !_tensor.is_dynamic() && _tensor.Batch().pad.before == 0) {
```

Comment:
What about fixing L579~L582 to use `offset` instead of `f_pad`? I think that will be (slightly) better in performance.

Reply:
Fixed